### PR TITLE
change the rest resp as json format

### PR DESCRIPTION
**Pull request by archerwen**
Based on [Issue#167](https://github.com/DMTF/python-redfish-library/issues/167), here are the changes.

### CODE REVIEW

File: src/redfish/rest/v1.py [hunk 1]
```diff
@@ -951,11 +951,11 @@ def _rest_request(self, path, method='GET', args=None, body=None,
 
                     try:
                         LOGGER.debug('HTTP RESPONSE for %s:\nCode: %s\n\nHeaders:\n' \
-                                 '%s\nBody Response of %s: %s\n'%\
+                                 '%s\nBody Response of %s:\n%s\n'%\
                                  (restresp.request.path,
                                 str(restresp._http_response.status_code)+ ' ' + \
                                 restresp._http_response.reason,
-                                headerstr, restresp.request.path, restresp.read))
+                                headerstr, restresp.request.path, json.dumps(json.loads(restresp.read), indent=4)))
```
Comment (mraineri) [inline src/redfish/rest/v1.py:958]: Could we have it dynamically check if the response body is JSON-encoded and fall back on the old method if it's not JSON?

Comment (archerwen) [reply src/redfish/rest/v1.py]: I've updated, do you think is this ok?

Comment (mraineri) [reply src/redfish/rest/v1.py]: Looks good to me! Thanks!

**Comment by mraineri** [comment]
Will see what others think; thanks for the change!

**Comment by archerwen** [comment]
Sure, welcome!!

**Comment by mraineri** [comment]
Approving out of cycle to get this added

**Pull request merged**